src/Components/App.tsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
+import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
```

Comment:
Why do you add this import of you're not using the imported objects?

Reply:
I must have missed that. I create tslint rule for unused variables plus I added husky to check this kind of thing before commit (also tests).